src/Keboola/StorageApi/Client.php

```diff
@@ -421,6 +421,23 @@ public function listSharedBuckets($options = [])
         return $this->apiGet($url);
     }
 
+    public function getSharedBucketDetail($projectId, $bucketId, $options = [])
```

Comment:
Je tohle shared bucket detail? Není to shared bucket tables?

Reply:
tj. myslel bych si, že by se měla změnit ta URL a tohle zůstat shared bucket detail

Reply:
Jo mas pravdu, tu url som upravil jak v klientovy tak v connection